x-pack/platform/plugins/shared/stack_alerts/public/rule_types/es_query/expression/esql_query_expression.tsx

```diff
@@ -61,7 +61,7 @@ const getWarning = (duplicateAlertIds: Set<string>, longAlertIds: Set<string>) =
   if (duplicateAlertIds.size > 0) {
     return i18n.translate('xpack.stackAlerts.esQuery.ui.alertPerRowWarning', {
       defaultMessage:
-        'Your alerts do not appear to be unique, which will delay recovery of your alerts.',
+        'Your alerts do not appear to be unique, we noticed multiple rows with the same alert ID.',
```

Comment:
Dang, seems like "we" doesn't work well here.  

Maybe

- 'Some rows generated the same alertID:'

I still have a feeling we may end up not wanting to display this "warning", but I guess we'll see!

Reply:
We can remove it, if you think that's better. I don't feel strongly either way about keeping the warning, bc I could see it being confusing and you're right that it is probably hard to get duplicate alert ids

Reply:
I think if we're going to return a warning during execution, it would make sense to show the warning if we can here. Maybe something less first-person based like `Test returned multiple rows with the same alert ID. Consider grouping on a different field`? Or something actionable?

Reply:
I updated the warning message in this commit [4ecd058](https://github.com/elastic/kibana/pull/212135/commits/4ecd058fdcd74d4e2f0fb777de79587933aba71b)